fashify/src/App.js

```diff
@@ -1,8 +1,8 @@
-import { Route, Routes, useLocation } from "react-router-dom";
+import { Navigate, Route, Routes, useLocation } from "react-router-dom";
 
 import Mockman from "mockman-js";
 import { Nav } from "./components";
-import { Home, ProductDetails, ProductList } from "./pages";
+import { Home, ProductDetails, ProductList, WishList } from "./pages";
 import Login from "./pages/Auth/Login/Login";
 import { useAuth } from "./contexts/auth-context";
```

Comment:
Suggestion: can use barrel export

so that you can directly import from `../contexts`